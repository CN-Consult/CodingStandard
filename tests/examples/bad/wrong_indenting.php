<?php
/**
 *
 * @file
 * @version 2.11
 * @copyright 2020 CN-Consult GmbH
 * @author Daniel Haas <daniel.haas@cn-consult.eu>
 */

/**
 * Testfunction
 */
function testIndent()
{
    echo "this line is indented with space instead of tab!";
  $a = 1;
    if ($a)
    {
      echo "this is not indented enough!";
    }
}