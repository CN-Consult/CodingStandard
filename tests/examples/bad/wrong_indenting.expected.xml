<?xml version="1.0" encoding="UTF-8"?>
<phpcs version="does not matter">
    <file name="examples/bad/wrong_indenting.php" errors="2" warnings="0" fixable="2">
        <error line="16" column="3" source="Generic.WhiteSpace.ScopeIndent.Incorrect" severity="5" fixable="1">Line indented incorrectly; expected at least 1 tabs, found 0</error>
        <error line="19" column="7" source="Generic.WhiteSpace.ScopeIndent.Incorrect" severity="5" fixable="1">Line indented incorrectly; expected at least 2 tabs, found 1</error>
    </file>
</phpcs>
